utils/permissions/src/permissions.mjs

```diff
@@ -0,0 +1,130 @@
+import {
+  check,
+  request,
+  RESULTS as results,
+  checkNotifications as rnpCheckNotifications,
+  PERMISSIONS as rnpPermissions,
+  requestNotifications as rnpRequestNotifications,
+} from 'react-native-permissions'
+import { is } from '@ma-shop/is'
+
+
+function getMotionPermission () {
+  let permission = rnpPermissions.ANDROID.ACTIVITY_RECOGNITION
+  if (is.ios()) {
+    permission = rnpPermissions.IOS.MOTION
+  }
+
+  return permission
+}
+/// @name getLocationPermission
+/// @author Matt Murtagh
+/// @description
+/// Gets the appropriate location permission depending on OS
+/// @returns {string} - the OS specific location permission
+function getLocationPermission () {
+  let permission = rnpPermissions.ANDROID.ACCESS_FINE_LOCATION
+  if (is.ios()) {
+    permission = rnpPermissions.IOS.LOCATION_ALWAYS
+  }
+
+  return permission
+}
+
+export const permissions = {
+  /// @name requestNotifications
+  /// @author Matt Murtagh
+  /// @description
+  /// Requests the user for permission to send notifications.
+  /// @returns {promise} - promise that resolves to a string representation of the
+  /// notification permission (unavailable, denied, blocked, granted)
+  async requestNotifications () {
+    const { status } = await rnpRequestNotifications([ 'alert' ])
```

Comment:
This function is similar in structure to `checkNotifications()`, but the format for the return is different. I would say make them match each other (up to you on which format you stick with)